data/174/627/716/1/1746277161.geojson

```diff
@@ -37,9 +40,15 @@
     "wof:country":"MX",
     "wof:created":1630532174,
     "wof:geomhash":"13b4fdabaf833f229fef0bcbd596cfc8",
-    "wof:hierarchy":[],
+    "wof:hierarchy":[
+        {
+            "campus_id":1746277161,
+            "continent_id":102191575,
+            "country_id":85633293
```

Comment:
Shouldn't these be getting at least a region in their hierarchy? If not a `county` or `locality` as well?

Reply:
So with each of these campus PRs (US, MX, CA, soon to be AU), I'm manually curating hierarchies for these places. I'm doing this for two reasons.. 1. the PIP process on some of these geometries (especially Canada) could take days to complete and 2. these are federally managed places that often span more than one region.. WOF's PIP tools couldn't handle multiple hierarchies right now.

So I've added campus, continent, and country placetypes to each campus hierarchy. I thought about adding a `wof:controlled` property to some of these, but I don't think we'd want to lock these hierarchies from future updates... does this manual approach work?

Reply:
I see. Since it's what you already did in USA sure?

Seems like we need a better long term solution for PIPing...

Reply:
Agreed.. I think this solution is temporary to get these features into WOF, but longer term it'd be nice to review the logic in WOFs PIP tools.